src/Commands/User.php

```diff
@@ -299,7 +302,12 @@ private function create(?string $username = null, ?string $email = null): void
         $userModel = model(UserModel::class);
 
         $user = new UserEntity($data);
-        $userModel->save($user);
+
+        if ($username === null) {
+            $userModel->allowEmptyInserts()->save($user);
+        } else {
+            $userModel->save($user);
+        }
 
         $this->write('User "' . $username . '" created', 'green');
```

Comment:
If there is no username, the appropriate message will not be printed. Please fix this issue.

Reply:
How?

Reply:
@kenjis Currently, the message output is as follows.
`User "" created`
Maybe message `New user created.` is enough.

Reply:
```php
        if ($username === null) {
            $userModel->allowEmptyInserts()->save($user);
            $this->write('New User created', 'green');
        } else {
            $userModel->save($user);
            $this->write('User "' . $username . '" created', 'green');
        }
```

Reply:
Done.